src/components/VoteButton.jsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+import * as api from '../utils/Api.js';
+
+const VoteButton = ({ articleId, setArticle, addedVotes, setAddedVotes }) => {
+
+    const id = articleId.article_id;
+    const [err, setErr] = useState("")
+
+    const handleClick = () => {
+        if (addedVotes === 0) {
+            setAddedVotes(1);
+            api.voteArticle(id, 1).catch(() => {
+                setErr("something went wrong, try again later")
+                setAddedVotes(0)
+            })
+
+        }
+
+        // None DRY way of updating article votes
+        // setArticle((currentArticle) => {
+        //     return { ...currentArticle, votes: currentArticle.votes + 1 }
+        // })
+        // api.voteArticle(id)
+        //     .then((updatedArticle) => {
+        //     }).catch(() => {
+        //         setArticle((currentArticle) => {
+        //             return { ...currentArticle, votes: currentArticle.votes - 1 }
+        //         })
+        //         setErr("something went wrong, try again later")
+        //     })
+    }
```

Comment:
Could you get rid of any comments. 
Comments are one of the worst to leave in a project as no one else will delete them as they presume someone else needs them, or they're there for a reason